lib/zewo.rb

```diff
@@ -341,8 +310,13 @@ def checkout()
 
         if options[:branch]
           matched = options[:branch]
+          matched = `cd #{repo.name} && git tag`
+                    .split("\n")
+                    .select { |t| t.start_with?(options[:tag]) }
+                    .last
         end
 
+        matched = options[:branch] if options[:branch]
         if matched
```

Comment:
I don't think I understand this part. why do you assign `matched = options[:branch]` and then later

```
matcher = `cd #{repo.name} && git tag`
                     .split("\n")
                     .select { |t| t.start_with?(options[:tag]) }
                     .last
```
 this makes `matched = options[:branch]` kinda useless doesn't it?. and later you do `matched = options[:branch] if options[:branch]` which makes the whole code before useless too, right? am I crazy? haha

Also I sent you a DM in Zewo's slack. You should check later! (:
Again, thanks a lot for this PR.